JavaScript/JavaScriptSDK.Tests/CheckinTests/Initialization.tests.ts

```diff
@@ -30,7 +30,8 @@ class InitializationTests extends TestClass {
             disableAjaxTracking: true,
             overridePageViewDuration: false,
             maxAjaxCallsPerView: 44,
-            disableDataLossAnalysis: true
+            disableDataLossAnalysis: true,
+            disableCorrelationHeaders: false
```

Comment:
Because, like CORS in general, there might be less common cases where adding our headers will break customer's server side (e.g. if they whitelist headers for security reason).
My plan is once we roll it out (and support in UI) we'll validate it first and then turn it ON by default.